CONTRIBUTING.md

```diff
@@ -139,6 +139,25 @@ This package has a `src/agoradatatools/etl/transform` submodule.  This folder ho
     - Use `pytest.mark.parameterize` to loop through multiple datasets in a single test.
     - The class `TestTransformGenesBiodomains` can be used as an example for future tests contibuted.
 
+### Great Expectations
+
+This package uses [Great Expectations](https://greatexpectations.io/) to validate output data.  The `great_expectations` folder houses our file system data context and Great Expectations-specific configuration files. Eventually, our goal is for each `agora-data-tools` dataset to be convered by an expectation suite. To add data validation for more datasets, follow these steps:
+
+1. Create a new expectation suite by defining the expectations for the new dataset in a Jupyter Notebook inside the `gx_suite_definitions` folder. Use `metabolomics.ipynb` as an example.
+1. Run the notebook to generate the new expectation suite. It should populate as a JSON file in the `great_expectations/expectations` folder.
+1. Add support for running  Great Expectations on a dataset by adding the `gx_folder` key to the configuration for your datatset in both `test_config.yaml` and `config.yaml`. The `gx_folder` should be a Synapse ID pointing to the folder where we will upload generated HTML reports from Great Expectations. If a folder specific to your dataset does not yet exist in the proper locations ([Prod](https://www.synapse.org/#!Synapse:syn52948668), [Testing](https://www.synapse.org/#!Synapse:syn52948670)), create folders named after the dataset and copy the new folders' Synapse IDs to the config files.
```

Comment:
```suggestion
1. Add support for running Great Expectations on a dataset by adding the `gx_folder` key to the configuration for your datatset in both `test_config.yaml` and `config.yaml`. The `gx_folder` should be a Synapse ID pointing to the folder where we will upload generated HTML reports from Great Expectations. If a folder specific to your dataset does not yet exist in the proper locations ([Prod](https://www.synapse.org/#!Synapse:syn52948668), [Testing](https://www.synapse.org/#!Synapse:syn52948670)), create folders named after the dataset and copy the new folders' Synapse IDs to the config files.
```

Reply:
Just to be clear, you want people to create folders named after the dataset?

Reply:
Yes, there is a `Great Expectations Reports` folder that exists for both the prod and testing data already. Right now each of those folders have one child folder called `metabolomics`. For each subsequent dataset that we add GX support for, a new child folder with a name corresponding with the dataset name should be created, and the Synapse ID for that folder added as the `gx_folder` for that dataset in the configuration files.